tests/unit/stats/test_aggregation_data.py

```diff
@@ -1,4 +1,5 @@
 # Copyright 2018, OpenCensus Authors
+# Copyright 2018, OpenCensus Authors
```

Comment:
?

Reply:
Removed, seems to be a bad copy-and-paste.